run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -234,41 +249,125 @@ if [[ "x${dustDead}" == "xon" || "x${tomasDustDead}" == "xon" ]]; then
+            fi
+
+        #---------------------
+        # DustL23m + GEOS-IT
+        #---------------------
+        elif [[ "x${metField}" == "xGEOSIT" ]]; then
+            if [[ "x${CS_RES}" == "x30" ]]; then
+                Dust_SF=4.231E-03
+            elif [[ "x${CS_RES}" == "x48" ]]; then
+                Dust_SF=3.308E-03
+            elif [[ "x${CS_RES}" == "x90" ]]; then
+                Dust_SF=2.614E-03
+            elif [[ "x${CS_RES}" == "x180" ]]; then
+                Dust_SF=2.314E-03
+            else
+                msg="Dust mass tuning factors for ${CS_RES} are not available "
+                msg+="for GEOS-IT meteorology.  Please add the tuning factor "
+                msg+="you wish to use for this resolution."
+                echo "${msg}"
+                exit 1
+            fi
+    
+        #---------------------
+        # DustL23M + MERRA2
+        #---------------------
+        elif [[ "x${metField}" == "xMERRA2" ]]; then
+    	    if [[ "x${CS_RES}" == "x24" ]]; then
+                Dust_SF=7.944E-03
+            elif [[ "x${CS_RES}" == "x48" ]]; then
+                Dust_SF=4.253E-03
+            elif [[ "x${CS_RES}" == "x90" ]]; then
+		Dust_SF=4.341E-03
+            elif [[ "x${CS_RES}" == "x180" ]]; then
+		Dust_SF=3.932E-03
+            else
+                msg="Dust mass tuning factors for ${CS_RES} are not available "
+                msg+="for MERRA-2 meteorology.  Please add the tuning factor "
+                msg+="you wish to use for this resolution."
+                echo "${msg}"
+                exit 1
+            fi
+ 
+        else
+            msg="Dust mass tuning factors for ${CS_RES} are not available "
+            msg+="for ${metField} meteorology.  Please add the tuning factor "
+            msg+="you wish to use for this resolution."
+            echo "${msg}"
+            exit 1
+
+        fi
+	
+    #------------------------------------------------------------------------
+    # Dust mass tuning factors for TOMAS_DustDead extension are the same
+    # as were previously computed for the DustDead extension
+    #------------------------------------------------------------------------
+    else       
+        if [[ $CS_RES -eq 24 ]]; then
+            if [[ "x${metField}" == "xMERRA2" ]]; then
+                Dust_SF=5.4856e-5
+            elif [[ "x${metField}" == "xGEOSFP" ]]; then
+                Dust_SF=4.6495e-5
+            elif [[ "x${metField}" == "xGEOSIT" ]]; then
+                Dust_SF=3.7410e-5
+            else
+                msg="TOMAS DustDead scale factors are currenlty only "
+                msg+="available for C24 with MERRA-2, GEOSFP, or GEOSIT."
+                echo "${msg}"
+                exit 1
+            fi
+        #
+        # NOTE: Met-field dependent mass tuning factors for the grid
+        # resolutions listed below will be added later.
+        # Leave the prior values for now.
+        #
+        elif [[ $CS_RES -eq 30 ]]; then
```

Comment:
For this and all other resolutions where the old dust tuning factors are used there should be a warning printed to log.